docker-compose.sc.yml

```diff
@@ -0,0 +1,70 @@
+version: '2.1'
+services:
+  rubyrails:
+    container_name: "classroom_rubyrails"
+    build: 
+      context: .
+      dockerfile: Dockerfile-sc
+    volumes: 
+      - .:/usr/src/app
+      - /usr/src/app/.bundle
+      - /usr/src/app/vendor
+    ports: 
+      - "5000:5000"
+    entrypoint: bash
```

Comment:
the dev environment simply opens a shell. Users attach to it and run script/setup and script/server etc just as they would for a native installation.